en/docs/learn/configuring-identity-analytics.md

```diff
@@ -15,18 +15,51 @@ To do this, the developers need to view authentication statistics about the logi
 2. [Download WSO2 Identity Server Analytics distribution](https://github.com/wso2/analytics-is/releases/tag/v5.8.0-rc3).
 
 ## Enable analytics
+Follow the steps below to configure WSO2 Identity Server Analytics.
 
-Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server. 
+1.	Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server. 
  
-``` toml
-[identity_mgt.analytics_login_data_publisher]
-enable=true
-```
+    ``` toml
+    [identity_mgt.analytics_login_data_publisher]
+    enable=true
+    ```
+
+    ```toml
+    [identity_mgt.analytics_session_data_publisher] 
+    enable=true
+    ```
+
+
+2.  An HTTP connection is used to communicate between WSO2 IS and WSO2
+    IS Analytics. Therefore, you must add the certificate of WSO2 IS Analytics to WSO2 IS.
+    Follow the steps given below to import the certificate from WSO2 IS Analytics
+    to WSO2 IS. Import the
+    `           public certificate          ` of each keystore to the
+    `           client­-truststore.jks          ` of the WSO2 IS. This example uses the default keystores and
+    certificates.
+
+    1.  Navigate to the
+        `             <ISANALYTICS_HOME>/resources/security            `
+        directory on a new terminal window and run the following
+        command. The default keystore password is `wso2carbon`.
+
+        ``` java
+        keytool -export -alias wso2carbon -keystore wso2carbon.jks -file sp.cer
```

Comment:
Use the .pem as in this doc https://is.docs.wso2.com/en/latest/learn/using-wso2-is-analytics-for-adaptive-authentication/